recipe/deploy/check_for_known_bugs.php

```diff
@@ -0,0 +1,50 @@
+<?php
+/**
+ * Check hosts for known bugs.
+ *
+ * @author Eugene Dzhumak <elforastero@ya.ru>
+ */
+
+namespace Deployer;
+
+use Deployer\Exception\RuntimeException;
+use Deployer\Host\Localhost;
+
+task('check_for_known_bugs', function () {
+    $OSWithKnownBugs = [
+        'ubuntu 14.04'
```

Comment:
I was thinking of this before, but checking for OS version can lead to false positive. 
Such task should check in feature style (i.e. try reproduce this bug) as user can manually update coreutils and so on.